### PR TITLE
add favicon and correct title

### PR DESCRIPTION
**Pull request by sahsisunny**
# add favicon and correct title

# Issue #294

### What is the change?

- Add favicon file in images folder
- Add favicon link in head of all pages 
- Change title of all pages based on the page

### Is Development Tested?

- [x] Yes

### CODE REVIEW

**Comment by vinit717** [comment]
Can you add screenshot of it how it is going to look like now

**Comment by vinit717** [comment]
Can you add screenshot of it how it is going to look like now

**Comment by sahsisunny** [comment]
> Can you add screenshot of it how it is going to look like now

image

**Review by vinit717 — APPROVED** [review]

File: featureFlag/index.html [hunk 1]
```diff
@@ -3,9 +3,10 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="/images/index.ico" type="image/x-icon" />
     <link rel="stylesheet" href="../style.css" />
     <link rel="stylesheet" href="style.css" />
-    <title>Feature Flag</title>
+    <title>Feature Flag | Dashboard RDS</title>
```
Comment (ankushdharkar) [inline featureFlag/index.html:9]: Why does this title look different than patterns followed on other pages?

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: featureFlag/index.html (hunk 1 above)
Comment (sahsisunny) [reply featureFlag/index.html:9]: Means the full name of RDS should have been like 
`<title>Feature Flag | Dashboard Real Dev Squad</title>` ?

Comment (sahsisunny) [reply featureFlag/index.html]: have changed

**Pull request merged**